epysurv/models/timepoint/_base.py

```diff
@@ -36,9 +37,16 @@ class TimepointSurveillanceAlgorithm:
 
     def fit(self, data: pd.DataFrame) -> "TimepointSurveillanceAlgorithm":
         """Expects data with time series index, case counts and outbreak labels."""
-        # Remove outbreaks cases from baseline.
+        self._validate_data(data)
         data = data.copy()
-        data.n_cases -= data.n_outbreak_cases
+        if "n_outbreak_cases" in data.columns:
+            # Remove outbreaks cases from baseline.
+            data.n_cases -= data["n_outbreak_cases"]
```

Comment:
What if `n_cases` and `n_outbreak_cases` are already separated by the user? Then subtracting would falsify `n_cases`.

Reply:
The semantics of `n_cases` is the total number of cases (outbreak + non outbreak). So I do not see a problem here.

Reply:
Ah, I found it in the user guide.